src/plugin-slots/HomePromoVideoSlots/HomePromoVideoModalSlot/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { PluginSlot } from '@openedx/frontend-plugin-framework';
+
+import { VideoModal } from '@src/generic';
+import { HomePromoVideoModalContentSlot } from '../HomePromoVideoModalContentSlot';
+import { HomePromoVideoModalSlotProps } from './types';
+
+export const HomePromoVideoModalSlot = ({ isOpen, close, videoId }: HomePromoVideoModalSlotProps) => (
+  <PluginSlot
+    id="org.openedx.frontend.catalog.home_page.promo_video_modal"
+    slotOptions={{
+      mergeProps: true,
+    }}
+    pluginProps={{
+      isOpen,
+      close,
+      videoId,
+    }}
+  >
+    <VideoModal
+      isOpen={isOpen}
+      close={close}
+      pluginSlotComponent={
+        <HomePromoVideoModalContentSlot videoId={videoId} />
```

Comment:
I have doubts about where to pass the HomePromoVideoModalContentSlot correctly.
There are two options:
1. via the plugin slot (`HomePromoVideoModalSlot`)
2. directly from the [HomeBanner](https://github.com/openedx/frontend-app-catalog/pull/7/commits/be56d049238ca008174e6b68b74d586358feb063#diff-63458a4dc4488e32962289495575a7476ae66f161f2289b5627ff5749e648e30R52) component as a prop in the HomePromoVideoModalSlot

I chose the first option because I did not find any advantages in the second